src/cloudant/_common_util.py

```diff
@@ -234,6 +234,28 @@ def codify(code_or_str):
         return _Code(code_or_str)
     return code_or_str
 
+def get_docs(r_session, url, encoder=None, **kwargs):
```

Comment:
Any thoughts on renaming `kwargs` to `params` as described [here](https://github.com/cloudant/python-cloudant/pull/185#discussion_r69210554)?